examples/tornjak-keycloak-client-import.json

```diff
@@ -0,0 +1,93 @@
+{
+      
+      "clientId": "Tornjak-React-auth",
+      "surrogateAuthRequired": false,
+      "enabled": true,
+      "alwaysDisplayInConsole": true,
+      "clientAuthenticatorType": "client-secret",
+      "secret": "**********",
+      "redirectUris": [
+        "*"
+      ],
+      "webOrigins": [*],
+      "notBefore": 0,
+      "bearerOnly": true,
```

Comment:
also it gives error in browser when this is set to true - setting it to false, things work